apps/ledger-live-desktop/src/renderer/components/ToastOverlay/TimeBasedProgressBar.tsx

```diff
@@ -29,25 +22,22 @@ export function TimeBasedProgressBar({ duration, onComplete, nonce = 1 }: Props)
       clearTimeout(timeout);
     };
   }, [duration, onComplete, nonce]);
+
   return (
     <div
       style={{
         width: "100%",
         height: 5,
       }}
     >
-      {transitions.map(({ key, props }) => (
-        <animated.div
-          key={key}
-          style={{
-            ...props,
-            height: 5,
-            width: "100%",
-            transformOrigin: "left center",
-            background: "#8a80db", // "#00000022",
-          }}
-        />
-      ))}
+      <animated.div
+        style={{
+          ...style,
+          height: 5,
+          transformOrigin: "left center",
+          background: "#8a80db",
```

Comment:
you may need to use theme colors to handle dark/light mode.
This is a suggestion as i don't know if the toast need to be white on both cases.

Reply:
This colour is not in our DS so I assume it was a decision when the component has been created to have it like this

Reply:
Indeed I think it was the case Lucas, it currently looks okay in both themes so it's okay to keep this one like that but we won't do it this way for any new features

Reply:
sounds clear for me, thank you both